### PR TITLE
profile styling

### PR DESCRIPTION
**Pull request by samboonzaayer**

### CODE REVIEW

File: frontend/components/profile/ViewProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { View, Text, Button, Image, StyleSheet, Dimensions } from 'react-native';
```
Comment (boyiq) [inline frontend/components/profile/ViewProfile.jsx:2]: Need to import some elements from react native elements, add:
`import { Card, Header, Chip } from '@rneui/themed';`

File: frontend/components/profile/ViewProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      <Image style={{
+        alignSelf: 'center',
+        zIndex: 2,
+        width: 100,
+        height: 100,
+        borderRadius: 50,
+        margin: 10,
+        borderRadius: 20
+      }}
+      source={'https://www.akc.org/wp-content/uploads/2017/11/Longhaired-Dachshund-standing-outdoors.jpg'} />
```
Comment (boyiq) [inline frontend/components/profile/ViewProfile.jsx:25]: For the image to show up, source needs to be reformatted, see below: 
`source={{uri: 'https://www.akc.org/wp-content/uploads/2017/11/Longhaired-Dachshund-standing-outdoors.jpg'}}`

File: frontend/components/profile/ViewProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { View, Text, Button, Image, StyleSheet, Dimensions } from 'react-native';
+
+export default ViewProfile = ({ navigation }) => {
+  const dWidth = Dimensions.get('window').width;
+  const dHeight = Dimensions.get('window').height;
+
+  return (
+    <View style={styles.moreInfo}>
+      <Button title="Log Out" style={{
```
Comment (boyiq) [inline frontend/components/profile/ViewProfile.jsx:10]: react native <Buttons> are not flexible in styling, it won't reflect what's specified in style. Try substituting <Button> with <TouchableOpacity>. Any text on the button wrap it in a <Text> and put inside <TouchableOpacity>

File: frontend/components/profile/ViewProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { View, Text, Button, Image, StyleSheet, Dimensions } from 'react-native';
+
+export default ViewProfile = ({ navigation }) => {
+  const dWidth = Dimensions.get('window').width;
+  const dHeight = Dimensions.get('window').height;
+
+  return (
```
Comment (boyiq) [inline frontend/components/profile/ViewProfile.jsx:8]: Wrap everything returned in another <View> and set background to white.

File: frontend/components/profile/ViewProfile.jsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+          <Chip
+            title="People Friendly"
+            size="xs"
+            color="#FFD8A9"
+            containerStyle={{ padding: 2 }}
+            titleStyle={{ color: 'black' }}
+          />
+        </View>
+        <Text style={{ fontSize: 16, paddingLeft: 5 }}>This is a bio.</Text>
```
Comment (boyiq) [inline frontend/components/profile/ViewProfile.jsx:66]: Maybe write something fun here for the bio? :)

**Review by boyiq — CHANGES_REQUESTED** [review]
Reviewed the profile page. Will review login and signup when styling is done. A few changes and should be good to go

**Review by hecwon — CHANGES_REQUESTED** [review]
What Boyi said

**Pull request merged**